packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -42,7 +42,7 @@ export interface AccordionTitleProps
   index?: number
 
   /** Ref to the clickable element that contains the title. */
-  contentRef: React.Ref<HTMLElement>
+  contentRef?: React.Ref<HTMLElement>
```

Comment:
is it possible to also remove the `_.noop` from the tests?

Reply:
done, pls check https://github.com/stardust-ui/react/pull/1418/files#r288565651